db/seeds.rb

```diff
@@ -42,22 +54,38 @@ def generate_users
   end
 
   def generate_dogs
-    99.times do |i|
+    50.times do |i|
       dog = Dog.create!(
       user_id: i + 1 ,
-      name: dog_names.shuffle[0],
-      age: rand(10) + 1,
-      weight: rand(50) + 10,
+      name: male_names,
+      age: age,
+      weight: size,
       breed: ["lab", "poodle", "golden"].shuffle.first,
-      gender: ["male", "female"].shuffle.first,
-      street: Faker::Address.street_address,
-      city: Faker::Address.city,
-      state: Faker::Address.state,
-      zipcode: Faker::Address.zip_code,
+      gender: "Male",
+      street: "",
+      city: "Denver",
+      state: "C0",
+      zipcode: "8020" + (rand(9) + 1).to_s,
       img: "http://loremflickr.com/300/300/puppy"
       )
       puts "Dog #{i + 1}: #{dog.name} created!"
     end
+    48.times do |i|
+      dog = Dog.create!(
+      user_id: i + 51 ,
```

Comment:
Indent the first parameter one step more than the previous line.<br>Space found before comma.